lib/generators/utils/entry.js

```diff
@@ -59,18 +59,18 @@ module.exports = (self, answer) => {
 							validate
 						)
 					])
-				).then(propAns => {
-					Object.keys(propAns).forEach(val => {
+				).then(entryPropAnswer => {
+					Object.keys(entryPropAnswer).forEach(val => {
 						if (
-							propAns[val].charAt(0) !== "(" &&
-							propAns[val].charAt(0) !== "[" &&
-							propAns[val].indexOf("function") < 0 &&
-							propAns[val].indexOf("path") < 0 &&
-							propAns[val].indexOf("process") < 0
+							entryPropAnswer[val].charAt(0) !== "(" &&
+							entryPropAnswer[val].charAt(0) !== "[" &&
+							entryPropAnswer[val].indexOf("function") < 0 &&
+							entryPropAnswer[val].indexOf("path") < 0 &&
+							entryPropAnswer[val].indexOf("process") < 0
 						) {
-							propAns[val] = `"${propAns[val]}.js"`;
+							entryPropAnswer[val] = `\'${entryPropAnswer[val].replace(/"|'/g,"").concat(".js")}\'`;
```

Comment:
The quote sanitization won't affect `function` as entries as we have a check for it. 

https://github.com/webpack/webpack-cli/blob/8787043beac452f605dc57bfa20aa0c1a545d188/lib/generators/utils/entry.js#L36-L44